packages/amplify_authenticator/lib/src/utils/validators.dart

```diff
@@ -69,43 +68,43 @@ Function validateSignUpPassword = (BuildContext context) {
               .contains(PasswordPolicyCharacters.requiresLowercase) &&
           !_lowercase.hasMatch(password)) {
         passwordHints
-            .add('* ${_hintStrings.password_requires_lowercase(context)}');
+            .add('* ${_hintStrings.passwordRequiresLowercase(context)}');
       }
       if (passwordCharacters
               .contains(PasswordPolicyCharacters.requiresUppercase) &&
           !_uppercase.hasMatch(password)) {
         passwordHints
-            .add('* ${_hintStrings.password_requires_uppercase(context)}');
+            .add('* ${_hintStrings.passwordRequiresUppercase(context)}');
       }
       if (passwordCharacters
               .contains(PasswordPolicyCharacters.requiresNumbers) &&
           !_numeric.hasMatch(password)) {
-        passwordHints
-            .add('* ${_hintStrings.password_requires_numbers(context)}');
+        passwordHints.add('* ${_hintStrings.passwordRequiresNumbers(context)}');
       }
       // TODO: symbols regex does not handle single quotes, currently handling with separate check.
       if (passwordCharacters
               .contains(PasswordPolicyCharacters.requiresSymbols) &&
           !_symbols.hasMatch(password) &&
           !password.contains("'")) {
-        passwordHints
-            .add('* ${_hintStrings.password_requires_symbols(context)}');
+        passwordHints.add('* ${_hintStrings.passwordRequiresSymbols(context)}');
       }
     }
     return null;
```

Comment:
I think something got messed up when I merged this code... can you please replace `return null` in line 92 with 
`return passwordHints.isNotEmpty ? passwordHints.join('\n') : null`?

Reply:
Fixed 👍